include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -108,6 +121,12 @@ typedef struct _oe_sgx_enclave_properties
  * @param[in] CAPTURE_PF_GP_EXCEPTIONS If true, allows the enclave to capture
  * #PF and #GP exceptions if the CPU supports the feature. The setting is
  * ignored otherwise
+ * @param[in] CREATE_ZERO_BASE_ENCLAVE If true, allows enclave to be created
+ * with a base address of 0x0. Else, the usual enclave creation logic is
+ * followed. Currently available only for SGX, linux.
+ * @param[in] ENCLAVE_START_ADDRESS If CREATE_ZERO_BASE_ENCLAVE is set, the
+ * enclave image start address. Must be higher than value in
+ * /proc/sys/vm/mmap_min_addr.
```

Comment:
this line seems Linux specific, but this function is not Linux specific from my understanding.  Please update comment to either remove Linux details or else add equivalent Windows details.

Reply:
This feature is linux specific (only intel/linux-sgx has the new changes). I will update changelog and sample (#4026) to mention this clearly. If the user chooses these parameters, they'll see a failure like this:

: 2021-06-30T02:07:40+0000.238811Z [(H)ERROR] tid(0x7fdbcb234740) | enclave_create_ex function not found.
1: 2021-06-30T02:07:40+0000.238819Z [(H)ERROR] tid(0x7fdbcb234740) | enclave_create_ex() was not found in installed libsgx_enclave_common.dll.

Reply:
Update this doxygen to say so, i.e., on Windows this must be 0, or whatever,